about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css" as="styles">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+    @import url('https://fonts.cdnfonts.com/css/cocogoose');
+  </style>
+  <title>N'A.I'tion Evolution</title>
+</head>
+<body>
+  <div class="social-bar hidden">
+    <div class="container">
+      <ul class="social-bar__list flex">
+        <li class="social-bar__item"><a href="#"><img src="icons/facebook_icon.png" alt="Icon"></a></li>
+        <li class="social-bar__item"><a href="#"><img src="icons/twitter_icon.png" alt="Icon"></a></li>
+        <li class="social-bar__item"><a href="#">English</a></li>
+        <li class="social-bar__item"><a href="#">My Page</a></li>
+        <li class="social-bar__item"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+  </div>
+
+  <header id="header" class="header p-fixed hidden">
+    <div class="header__logo"><img src="images/naition_logo.jpg" alt="Logo"></div>
+    <nav class="nav-bar">
+      <ul class="nav-bar__list">
+        <div class="nav-bar__item nav-bar__item--button"><button class="nav-bar__button"><img src="icons/nav_bar_close.png" alt="Close"></button></div>
+        <div><hr><li class="nav-bar__item"><a href="/index.html">Home</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="/about.html">About</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">Program</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">Join</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">Sponsors</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">News</a></li></div>
+        <div><hr><li class="nav-bar__item"><a class="nav-bar__item--design" href="#">NE Campaign</a></li><hr></div>
```

Comment:
- [ ] I would kindly suggest you work on the navbar section of the of the project both on the `home page ` and `about page` as currently the border around the last link is not project designed. and i would also implore you to make sure the links on both the top navbar and bottom navbar aligned at the end.

| Your Design 👇 | Original Design 👇 |
| ------------- | ------------- |
| <img width="949" alt="head-proj" src="https://github.com/lRebornsl/capstone-project/assets/78634841/1aca9ccc-7047-4222-b122-d21ef2697d26">  | <img width="626" alt="image" src="https://github.com/lRebornsl/capstone-project/assets/78634841/ee96c0eb-2bf3-45f0-9e05-00e3adc5cdfa">|